get-started/3-setup-editor.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: page
+title: "Get Started: Setup Editor"
+permalink: /get-started/setup-editor/
+---
+
+Flutter is a flexible toolkit, which supports writing Flutter apps using a
```

Comment:
Maybe more simply: "You can build apps with Flutter using any text editor, with our command-line tools. However, we recommend using one of our editor plugins for an even better experience. With our editor plugins, you can code complete, etc etc etc. Follow the steps before to add an editor plugin for _____ or __________. If you want to use a different editor, that's OK, simply skip ahead to ___________."

Reply:
+1 to Seth's suggestion.

Reply:
Good suggestion, done